test/math_test.py

```diff
@@ -1141,6 +1141,20 @@ def test_project_v2_raises_if_other_is_not_iterable(self):
         # act / assert
         self.assertRaises(TypeError, v.project, other)
 
+    def test_clamp_mag_v2_max(self):
+        v1 = Vector2(7, 2)
+        v2 = v1.clamp_magnitude(5)
+        expected_v2 = Vector2(4.80762, 1.37361)
+        self.assertEqual(expected_v2.x, round(v2.x, 5))
+        self.assertEqual(expected_v2.y, round(v2.y, 5))
```

Comment:
It's better to use the `assertAlmostEqual` function here, and everywhere else in the remaining `clamp_magnitude` tests